articles/install-python-qdk.md

```diff
@@ -21,20 +21,39 @@ Learn how to install the Quantum Development Kit (QDK) to develop Python host pr
 
 1. Install [Miniconda](https://docs.conda.io/en/latest/miniconda.html) or [Anaconda](https://www.anaconda.com/products/individual#Downloads). **Note:** 64-bit installation required.
 
-1. Open an Anaconda Prompt.
+1. (Windows) Open an Anaconda Prompt.
```

Comment:
So here we lead Windows users to use the Anaconda Prompt, and others OS to use conda in their terminal. But in the "Note" below there are instructions on how to run PowerShell in Windows. I know that they only put this into practice in case they want to use the terminal in VSCode, but for Windows users this might be confusing. 

Maybe we can ask Windows users to open a cmd prompt in VS Code so they don't need to do extra steps.

Reply:
Yeah, I have been thinking about the same thing.

I mean we could move the instructions about getting conda to work with PowerShell to the the IDE section, but the problem is it doesn't only apply to working with VS Code, but also with PowerShell itself. Plus it's only relevant if the user installed with conda.

Alternatively, in the conda tab, we could include steps in the main instructions to set up conda for all shells/systems, instead of using Anaconda Prompt on Windows/setting up conda with a shell on Linux/using a separate note box for PowerShell.

Or, we remove all instructions pertaining to the set up of a different shell on windows, but I think this will do users a disservice. And we need those instructions anyways on Linux. (I think this coincides with your suggestion in your last sentence, using only Anaconda Prompt on Windows and CMD if using VS Code. Note that this requires that user has conda in their path, or they'll always need to cd to the install location whenever they run a conda command, or install the conda start up script for CMD just like they would for PowerShell.)

Reply:
> And we need those instructions anyways on Linux. (I think this coincides with your suggestion in your last sentence, using only Anaconda Prompt on Windows and CMD if using VS Code. Note that this requires that user has conda in their path, or they'll always need to cd to the install location whenever they run a conda command, or install the conda start up script for CMD just like they would for PowerShell.)

I think this gets at the root of the problem: telling Windows users  to use `cmd` won't solve the issue, as the Anaconda installer for Windows defaults to only making the `conda` command available through the "Anaconda Command Prompt" or "Anaconda PowerShell" items on the Start Menu. Overriding that default when installing (using the add to PATH option) has a lot of surprising consequences, such that I don't think we should encourage doing so in our instructions.

If we want to make sure  that `conda` is available to VS Code users, telling them to open an Anaconda Command Prompt or Anaconda PowerShell session once and running `conda init powershell` (cmd is included by default, so that should get both) is needed, I think.

Reply:
I gave it another go which hopefully simplifies the presentation while still providing all the information needed. Let me know what you think, the preview has also been updated.